eidpauth/src/main/java/uk/ac/ucl/eidp/auth/jwt/JwtConfig.java

```diff
@@ -0,0 +1,58 @@
+package uk.ac.ucl.eidp.auth.jwt;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.security.Key;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.UnrecoverableEntryException;
+import java.security.cert.CertificateException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.annotation.PostConstruct;
+import javax.annotation.Resource;
+import javax.enterprise.context.ApplicationScoped;
+
+/**
+ * Configuration object for JWT.
```

Comment:
\[PMD\] INFO: Avoid unnecessary constructors \- the compiler will generate these for you